wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift

```diff
@@ -665,6 +666,7 @@ extension WireCallCenterV3 {
 
                         do {
                             for try await conferenceInfo in onConferenceInfoChange {
+                                try Task.checkCancellation()
```

Comment:
What doe this do?

Reply:
if the Task is cancel see `snapshot?.updateConferenceInfoTask?.cancel()` it will stop the execution of the task at that point. Otherwise cancel does nothing